Add 2 new icons and 1 new componentinfo

davidjosefson opened the pull request.

flameshikari commented:
Hello! There are some issues with your icons:

- `bank_id_alt_1`: square template is deformed

![image](https://github.com/user-attachments/assets/0a0b3fee-ada6-43be-a601-e11fb4368218)

- `lansforsakringar`: use Coral pink <code>#FF837D</code> as default red [according to the contribution guidelines](https://github.com/Delta-Icons/android/blob/master/.github/CONTRIBUTING.md#rules-2) and try to make straight figures without any extra anchors that makes them curvy

![image](https://github.com/user-attachments/assets/253f4c6a-a6e6-4b6a-9676-04222c514798)

Some tips to improve icons:
- `bank_id_alt_1`: remove the text to make the icon more simpler, the logo still makes the icon recognizable
- `lansforsakringar`: adapt the icon to the square template (it will be kind of free-form icon) and change blue color to Dark pastel blue `#728DBE`

![image](https://github.com/user-attachments/assets/66bda6db-eb51-4acf-9d10-28070151577d)

davidjosefson commented:
Awesome feedback! This is my first time doing anything with svgs and InkScape, still learning :)

bank_id:
- fixed the deformed square
- removed the text

lansforsakringar:
- changed blue and red colors
- removed all unnecessary anchors and made straight lines
- I **didn't** change it to a free-form square icon, since I think this icon and its colors need a white background to be usable

flameshikari commented:
Thanks for fixes! And welcome aboard!

I've made some fixes before merging. Keep the icon position in the center (`lansforsakringar` was shifted a little). And sometimes the icon size becomes float (176px -> 176.01px), maybe due export misconfiguration or accidental resizing. This isn't critical, but it can result in blurry edges on your PNG, so keep that in mind.
